orcoursetrion/actions/github.py

```diff
@@ -93,6 +94,35 @@ def rerun_studio(course, term, new_term, description=None):
     return repo
 
 
+def release_studio(course, term):
+    """Moves a studio course to be ready for production.
+
+    Currently this will just add a hook to the production server, but
+    it will eventually take care of everything else needed for a
+    transfer as well.
```

Comment:
Agreed - if a bunch of stuff doesn't end up being the same here in the future, abstracting it out into a function would literally be replacing one function call with another that looks the same. ;)